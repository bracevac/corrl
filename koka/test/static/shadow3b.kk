// shadow definition
function id(x) {
  function id2(x) -> x
  function id(y) -> id2(y)
  id(x)
}
