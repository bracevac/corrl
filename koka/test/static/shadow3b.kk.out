test/static/shadow3b.kk(3,16): warning: x shadows an earlier local definition or parameter
id : forall<a> (x : a) -> a